acceptance/openstack/mrs/v1/clusters_test.go

```diff
@@ -0,0 +1,141 @@
+package v1
+
+import (
+	"testing"
+
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/clients"
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/tools"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/mrs/v1/cluster"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/networking/v1/subnets"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/networking/v1/vpcs"
+	th "github.com/opentelekomcloud/gophertelekomcloud/testhelper"
+)
+
+func TestMrsClusterLifecycle(t *testing.T) {
+	client, err := clients.NewMrsV1()
+	th.AssertNoErr(t, err)
+
+	cc, err := clients.CloudAndClient()
+	th.AssertNoErr(t, err)
+
+	az := clients.EnvOS.GetEnv("AVAILABILITY_ZONE")
+	if az == "" {
+		az = "eu-de-02"
+	}
+
+	networkID := clients.EnvOS.GetEnv("NETWORK_ID")
+	vpcID := clients.EnvOS.GetEnv("VPC_ID")
+	keyPairName := clients.EnvOS.GetEnv("KEYPAIR_NAME")
+	if networkID == "" || vpcID == "" || keyPairName == "" {
+		t.Skip("OS_NETWORK_ID, OS_VPC_ID or OS_KEYPAIR_NAME env vars are missing but MRS Cluster test requires")
+	}
+
+	nwV1Client, err := clients.NewNetworkV1Client()
+	th.AssertNoErr(t, err)
+
+	vpc, err := vpcs.Get(nwV1Client, vpcID).Extract()
+	th.AssertNoErr(t, err)
+
+	subnet, err := subnets.Get(nwV1Client, networkID).Extract()
+	th.AssertNoErr(t, err)
+
+	name := tools.RandomString("mrs-create-", 3)
+	createOpts := cluster.CreateOpts{
+		BillingType:        12,
+		DataCenter:         cc.RegionName,
+		MasterNodeNum:      2,
+		MasterNodeSize:     "c3.xlarge.4.linux.mrs",
+		CoreNodeNum:        3,
+		CoreNodeSize:       "c3.xlarge.4.linux.mrs",
+		AvailableZoneID:    az,
+		ClusterName:        name,
+		Vpc:                vpc.Name,
+		VpcID:              vpc.ID,
+		SubnetID:           subnet.NetworkID,
+		SubnetName:         subnet.Name,
+		ClusterVersion:     "MRS 2.1.0",
+		ClusterType:        0,
+		VolumeType:         "SATA",
+		VolumeSize:         100,
+		SafeMode:           1,
+		ClusterAdminSecret: "Qwerty123!",
+		LoginMode:          1,
+		NodePublicCertName: keyPairName,
+		LogCollection:      1,
+		ComponentList: []cluster.ComponentOpts{
+			{
+				ComponentName: "Presto",
+			},
+			{
+				ComponentName: "Hadoop",
+			},
+			{
+				ComponentName: "Spark",
+			},
+			{
+				ComponentName: "HBase",
+			},
+			{
+				ComponentName: "Hive",
+			},
+			{
+				ComponentName: "Hue",
+			},
+			{
+				ComponentName: "Loader",
+			},
+			{
+				ComponentName: "Tez",
+			},
+			{
+				ComponentName: "Flink",
+			},
```

Comment:
Maybe add some function like `ExpandComponent` converting str slice to `ComponentOpts` slice? In most cases, we do this in the provider, but it can be done in the SDK too.